db/migrate/20231004220456_create_likes.rb

```diff
@@ -0,0 +1,12 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+     # t.integer :user_id
+     # t.integer :post_id
```

Comment:
- [ ] [OPTIONAL] Kindly remove the unused code in order to have a clean code base.